README.md

```diff
@@ -1,13 +1,12 @@
-# Гайд по I курсу
+<h1 align="center"> Гайд по I курсу 🐣 </h1>
 
-## Описание
+# Описание
 
 > Вашему вниманию представляется сборник мануалов и гайдов по первому курсу ВТ. Была проделана большая работа как тех, кто находил эту информацию, так и тех, кто структурировал, оформлял и объеденял в один источник. Мы старались :smile:
-<h4 align="center"> Первокурсники, пользуйтесь с умом и всем хороших оценок!)</h4>
 
-<a id="оглавление"></a>
-<details open>
-   <summary><h2><b> Оглавление </b></h2></summary>
+<p align="center"><b> Первокурсники, пользуйтесь с умом и всем хороших оценок!) </b></p>
```

Comment:
why deleting <h4> and replacing it with <p> ?
it's basically the same, i like h4 even more, idk